samples/isilon_v260_k8s_123.yaml

```diff
@@ -143,18 +143,12 @@ spec:
       # Allowed values: map of key-value pairs
       # Default value: None
       nodeSelector:
-      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
-      #  node-role.kubernetes.io/master: ""
```

Comment:
K8s 1.23 has examples for master under node selector and tolerations. Please revert the changes made in this file wrt examples for master under node selector and tolerations